src/pages/Home/Home.tsx

```diff
@@ -3,15 +3,7 @@ import Layout from "../../components/Layout/Layout";
 const Home = () => {
   return (
     <Layout>
-      <h1>Page Heading</h1>
-      <h2>Section Heading</h2>
-      <h3>Panel Heading</h3>
-      <p>
-        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Non error
-        dolor in nostrum minima odio a fuga saepe soluta adipisci perspiciatis
-        maxime tempora, temporibus beatae voluptas repellat rerum. Dignissimos,
-        necessitatibus.
-      </p>
+    
```

Comment:
If you remove this it gives an error in the console on my side. Can you put the dummy text back in?